cnf-certification-test/suite_test.go

```diff
@@ -142,18 +142,19 @@ func TestTest(t *testing.T) {
 	// Diagnostic functions will run when no focus test suites or labels are provided.
 	var diagnosticMode bool
 	if len(ginkgoConfig.FocusStrings) == 0 && ginkgoConfig.LabelFilter == "" {
+		log.Infof("TNF will run in diagnostic mode so no test case will be launched.")
 		diagnosticMode = true
 	}
 
 	// Set clientsholder singleton with the filenames from the env vars.
 	_ = clientsholder.GetClientsHolder(getK8sClientsConfigFileNames()...)
 
 	// Deploy the daemonset before getting the environment for the first time
-	if !diagnosticMode {
-		err := daemonset.DeployPartnerTestDaemonset()
-		if err != nil {
-			log.Errorf("Error deploying partner daemonset %s", err)
-		}
+	err := daemonset.DeployPartnerTestDaemonset()
+	if err != nil {
+		log.Errorf("Error deploying partner daemonset %s", err)
```

Comment:
Add vertical space before the comment.

Reply:
Done, but, please keep in mind we're not enforcing any strict coding/cosmetics guidelines on this project. 

Personally, for the sake of code readability, and as a general rule, also like to add those empty lines, but I don't see how an extra empty line can help readability in this particular code block, (`if {}`), that only takes 3 lines, where one of them is a comment. 

Also, notice that those extra empty-lines do increase the line count, which would potentially result in a linter error (funlen).

Reply:
I understand, personally, I hate vertical spaces. The only vertical space I put is before any comment. If you want separate some code, it is better to write a function.